app/styles/artisans/levelTasksView.sass

```diff
@@ -0,0 +1,12 @@
+#level-tasks-view
```

Comment:
File should be `level-tasks-view.sass`, and `courseGearView.sass` should be `course-gear-view.sass`, or deleted since it's empty.